### DISCUSSION TITLE
Automatic Brightness/ColorTemp

### DISCUSSION THREAD

**Discussion by chasehobbs** [Foddy/node-red-contrib-huemagic#281] [Q&A]
The new auto setting added to lights and groups for brightness and color temperature says it is based on the current time. I cannot find anything saying what settings correspond to what time. Is this also configurable anywhere?

**Comment by Foddy** [comment]
It is currently not configurable via the UI. You can find the corresponding function here:

https://github.com/Foddy/node-red-contrib-huemagic/blob/6c2a51a5d125210aa4ef069099d4599fc798ee10/huemagic/utils/color.js#L355-L367

The brightness is based on the current color temperature which is based on the current time. When the current color temperature is 300 or greater, the lights will get darker until the color temperature of 500 is reached or the brightness of 20% (minimum).

https://github.com/Foddy/node-red-contrib-huemagic/blob/6c2a51a5d125210aa4ef069099d4599fc798ee10/huemagic/hue-light.js#L370-L373

I am not quite sure how to make these methods configurable. If you have any idea, please let me know.
